### PR TITLE
updating classifications checkmark

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
checkmark to green box for classifications

### CODE REVIEW

File: src/sass/_profile.scss [hunk 1]
```diff
@@ -120,12 +120,23 @@
...
+        width: 30px;
+        margin: 10px 10px 10px 0;
+      }
+      > div {
+        display: inline-block;
+        vertical-align: middle;
+      }
     }
 
+    //Do we use the Updates component anywhere? (if not, i can delete lines 139-149)
```
Comment (elizabeth-jimenez) [inline src/sass/_profile.scss:139]: 💭

Comment (mjoyce91) [reply src/sass/_profile.scss:139]: 🤔  not sure

**Review by mjoyce91 — APPROVED** [review]

File: src/sass/_profile.scss (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/sass/_profile.scss:139]: 🤷‍♀️ 🤷 🤷‍♂️

**Comment by elizabeth-jimenez** [comment]
cancelled test to allow other PRs to merge first (concurrency limit was hit)

**Pull request merged**